client/my-sites/sites/sites.jsx

```diff
@@ -93,4 +108,13 @@ export default React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => ( {
```

Comment:
>@aduth Is there a reason why we don't sync all sites in `state.sites` ?

There's some context at a previous effort to this point in #7775 . I don't like the duplication, and I'm still skeptical the duplication could be eliminated in a timely matter, but despite this I'd be receptive to the idea of doing it anyways. It most negatively impacts users with many sites, which is a small minority (primarily us as Automatticians 😄 ).